app/src/androidTest/java/ch/epfl/sdp/peakar/general/MainMenuActivityTest.java

```diff
@@ -58,6 +58,17 @@ public void cleanUp(){
         Intents.release();
     }
 
+    /* Test that pressing the ranking button changes view to RankingsActivity */
+    @Test
+    public void TestRankingsButton(){
+        Intents.release();
+        Intents.init();
```

Comment:
Should this be here?

Reply:
I think it should be in a BeforeClass as well